src/bidiMapper/modules/cdp/CdpTarget.ts

```diff
@@ -437,13 +441,13 @@ export class CdpTarget {
   }
 
   async #toggleNetwork(enable: boolean): Promise<void> {
-    this.#networkDomainEnabled = enable;
+    this.#networkBidiDomainEnabled = enable;
     try {
       await this.#cdpClient.sendCommand(
         enable ? 'Network.enable' : 'Network.disable',
```

Comment:
could we force enable here instead of #unblock above?

Reply:
right, done